handsontable/src/plugins/filters/filters.js

```diff
@@ -138,7 +138,7 @@ export class Filters extends BasePlugin {
   constructor(hotInstance) {
     super(hotInstance);
     // One listener for the enable/disable functionality
-    this.hot.addHook('afterGetColHeader', (col, TH) => this.#onAfterGetColHeader(col, TH));
+    this.hot.addHook('afterGetColHeader', (col, TH, headerLevel) => this.#onAfterGetColHeader(col, TH, headerLevel));
```

Comment:
I'd simplify this to:
```suggestion
    this.hot.addHook('afterGetColHeader', (...args) => this.#onAfterGetColHeader(...args));
```